src/transport/raw/TCP.cpp

```diff
@@ -616,36 +620,17 @@ CHIP_ERROR TCPBase::TCPConnect(const PeerAddress & address, Transport::AppTCPCon
 
 void TCPBase::TCPDisconnect(const PeerAddress & address)
 {
-    CHIP_ERROR err = CHIP_NO_ERROR;
     // Closes an existing connection
     for (size_t i = 0; i < mActiveConnectionsSize; i++)
     {
         if (mActiveConnections[i].IsConnected())
         {
-            Inet::IPAddress ipAddress;
-            uint16_t port;
-            Inet::InterfaceId interfaceId;
+            const Inet::IPAddress & ipAddress = mActiveConnections[i].mPeerAddr.GetIPAddress();
+            uint16_t port                     = mActiveConnections[i].mPeerAddr.GetPort();
 
-            err = mActiveConnections[i].mEndPoint->GetPeerInfo(&ipAddress, &port);
```

Comment:
The commit message (and PR description) should explain why this change is being made.  Is the peer info in `mEndpoint` the same as `mPeerAddr`?  If they are, this is just code simplification, I guess? If they are not, why are we using one instead of the other?

This all needs to be in the commit message.

Reply:
In a sense, this is a simplification. The IPAddress and Port in mEndPoint is the same as that in mPeerAddr inside the corresponding ActiveConnection object which is readily available. Will add to the comments.